packages/workflow/src/Interfaces.ts

```diff
@@ -1250,6 +1250,16 @@ export interface INodeType {
 	};
 }
 
+/**
+ * This class serves as the base for all nodes using the new context API
+ * having this as a class enables us to identify these instances at runtime
+ */
+export abstract class Node {
```

Comment:
Shouldn't Node implement INodeType ?

Reply:
no. I'm trying to get rid of `INodeType`, and plan to hopefully move all nodes to extend from this base class instead.